package-scripts/datadog-agent/postrm

```diff
@@ -4,12 +4,17 @@ LINUX_DISTRIBUTION=$(grep -Eo "(Debian|Ubuntu|RedHat|CentOS|openSUSE|Amazon)" /e
 if [ -f /etc/debian_version ] || [ "$LINUX_DISTRIBUTION" == "Debian" ] || [ "$LINUX_DISTRIBUTION" == "Ubuntu" ]; then
     set -e
     if [ "$1" = purge ]; then
-        update-rc.d datadog-agent remove
+        if which systemd >/dev/null 2>&1; then
+	    systemctl remove datadog-agent
+	else
+	    update-rc.d datadog-agent remove
+        fi
         deluser dd-agent
         rm -rf /opt/datadog-agent
         rm -rf /var/log/datadog
         rm -rf /etc/dd-agent
         rm -rf /var/log/datadog
+	rm -rf /lib/systemd/system/datadog-agent.service
```

Comment:
This line has been removed anyway since this file is removed without our help in postrm